### ISSUE TITLE
View all gestures

### ISSUE THREAD

**Issue by 2br-2b** [moehriegitt/NeatLauncher#6]
It would be very nice if there was some way to view all the current gestures and be able to either reassign them or disable them from one screen

**Comment by moehriegitt** [comment]
Indeed.  I will think about this!  It would need to be a simple dialog, I did not like the complex and sometimes redundant ways of assigning gestures in other launchers. Maybe an overview to disabling (some of) them would already help?  Reassignment would then be done as it is done now: by first selecting the app.
